update metrics forwarder to support v2 (#601)
DataDog/datadog-operator: docs/how-to-contribute.md

```
### Deploy a basic `v2alpha1.DatadogAgent` resource.

Create a secret that contains an `api-key` and an `app-key`. By default the Operator is installed in the

> `system` namespace and only watches resources in the this namespace. As a result, the secret and deployment need to be done in the same namespace.

Reviewer: ```suggestion
`system` namespace, and only watches resources in this namespace. As a result, the secret and deployment must be within the same namespace.
```

> Then apply the `examples/v2alpha1/min.yaml` file which contains the mininum information needed to deploy the agent and related services.

Reviewer: ```suggestion
Then apply the `examples/v2alpha1/min.yaml` file which contains the mininum information needed to deploy the Agent and related services.
```
